static/src/javascripts/projects/common/utils/closest.js

```diff
@@ -0,0 +1,18 @@
+define(function () {
+    /**
+     * Adds a wrapper around Element.closest()
+     */
+    var matches = 'matches' in Element.prototype ? 'matches' : 'msMatchesSelector';
+    return 'closest' in Element.prototype ? closestNative : closestPolyfill;
+
+    function closestNative(element, selectors) {
+        return element.closest(selectors);
+    }
+
+    function closestPolyfill(element, selectors) {
+        while (element && !element[matches](selectors)) {
```

Comment:
Just tested it, it's fine, ignore the above. I thought `matches` was using a context, but it doesn't.